### DISCUSSION TITLE
Analyse code in github repo

### DISCUSSION THREAD

**Discussion by xeon123** [brianpetro/obsidian-smart-connections#507] [General]
I am trying to connect smart connection to my github project, so that it can analyze the code.
This project contains small scripts, but I would like to create some notes about it. Is it possible?

**Comment by Thomas-Richardson** [comment]
My intuition is that this isn't really a good idea. Nonetheless:
Smart connections is an obsidian plugin so only works on .md files in an obsidian vault.
To have it "see" and "understand" your project, you'd need to open obsidian and select the local copy of your repo as your vault (an Obsidian can literally treat any folder on your comp as a 'vault'). All scripts would have to be converted to .md files.
One way you could hack this is by maintaining a .md copy of every .py file. Obviously the .py files are the working code, but the .md copies are the files that obsidian (and smart connections) can "see"
